### PR TITLE
tools: syncer doc

### PR DESCRIPTION
**Pull request by GregoryIan**
@iamxy @holys @andelf PTAL

### CODE REVIEW

**Comment by ngaut** [comment]
Well done. Could you change "实例"  in the "syncer_sharding.png" to english? So we can unify the picture.

File: tools/syncer.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+pattern-table = "table_*"
+target-schema = "example_db"
+target-table = "table"
+```
+## 监控方案
+Syncer 使用开源时序数据库 Prometheus 作为监控和性能指标信息存储方案，使用 Grafana 作为可视化组件进行展示。
+
+Prometheus 是一个拥有多维度数据模型，灵活的查询语句的时序数据库。Prometheus 作为热门的开源项目，拥有活跃的社区及众多的成功案例。
+
+Prometheus 提供了多个组件供用户使用。目前，我们使用 Prometheus Server，来收集和存储时间序列数据。Client 代码库，在程序中定制需要的 Metrics 。Prometheus 主动抓取这些 Metrics, 配合 AlertManager 来实现报警机制。其结构如下图
```
Comment (andelf) [inline tools/syncer.md]: ...seems baidu translation.

File: tools/syncer.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+![](../media/syncer_monitor_scheme.png)
```
Comment (andelf) [inline tools/syncer.md]: should add figure title.
out latex/pdf generation use this info.

File: tools/syncer.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+[[route-rules]]
+pattern-schema = "example_db"
+pattern-table = "table_*"
+target-schema = "example_db"
+target-table = "table"
+```
+## 监控方案
+Syncer 使用开源时序数据库 Prometheus 作为监控和性能指标信息存储方案，使用 Grafana 作为可视化组件进行展示。
+
+Prometheus 是一个拥有多维度数据模型，灵活的查询语句的时序数据库。Prometheus 作为热门的开源项目，拥有活跃的社区及众多的成功案例。
```
Comment (andelf) [inline tools/syncer.md]: too many attribute clause. 
we Chinese never speak this.

File: tools/syncer.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+|  4 |    4 |
+|  5 |    5 |
++----+------+
+```
+
+`syncer` 每隔 30s 会输出当前的同步统计，如下
+
+
+```bash
+2016/10/27 15:22:31 syncer.go:668: [info] [syncer]total events = 1, insert = 1, update = 0, delete = 0, total tps = 0, recent tps = 0, binlog name = mysql-bin.000003, binlog pos = 1280.
```
Comment (holys) [inline tools/syncer.md]: Should update to the latest output

**Comment by holys** [comment]
Rest LGTM

File: tools/syncer.md (hunk 1 above)
Comment (GregoryIan) [inline tools/syncer.md]: 🤣  I would fix it, why not speak Chinese, It's Chinese document

**Comment by iamxy** [comment]
LGTM

**Pull request merged**